src/components/Tooltip/Tooltip.tsx

```diff
@@ -17,15 +18,24 @@ function customArrow(placement: Placement, arrowProps) {
   );
 }
 
-export interface TooltipProps extends CoreTooltipProps {}
+export interface TooltipProps extends CoreTooltipProps {
+  skin?: TooltipSkin;
+}
+
+export interface TooltipDefaultProps {
```

Comment:
I would go for
```
export type TooltipDefaultProps = Required<Pick<TooltipProps, 'skin'>>
```
So if you update `TooltipProps` you will get a compile error for the default props (forcing you to update it as well)

Reply:
Done